app/src/main/java/com/gh4a/utils/TaskUtils.java

```diff
@@ -0,0 +1,23 @@
+package com.gh4a.utils;
+
+import android.annotation.TargetApi;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+
+public class TaskUtils {
+    private TaskUtils() {
+    }
+
+    @TargetApi(Build.VERSION_CODES.LOLLIPOP)
+    public static void startNewTask(Context context, Intent intent) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            intent.addFlags(Intent.FLAG_ACTIVITY_NEW_DOCUMENT);
+            intent.addFlags(Intent.FLAG_ACTIVITY_MULTIPLE_TASK);
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
+                intent.addFlags(Intent.FLAG_ACTIVITY_LAUNCH_ADJACENT);
+            }
+        }
```

Comment:
Did you try what happens on API < 20 here? Also, no FLAG_ACTIVITY_NEW_TASK?

Reply:
Yeah I've tested it previously. As far as I remember older versions didn't have tasks feature so it always opened in the same task. For them we can remove that action from menu. I'm not sure about KitKat though. Will check again later.